Deprecated G4 calls in `GLG4PMTOpticalModel`

```
ratpac2/src/physics/src/GLG4PMTOpticalModel.cc:499:36: warning: ‘void G4FastStep::SetPrimaryTrackFinalTime(G4double)’ is deprecated: use ProposePrimaryTrackFinalTime instead [-Wdeprecated-declarations]
ratpac2/src/physics/src/GLG4PMTOpticalModel.cc:500:40: warning: ‘void G4FastStep::SetPrimaryTrackFinalMomentum(const G4ThreeVector&, G4bool)’ is deprecated: use ProposePrimaryTrackMomentumDirection instead [-Wdeprecated-declarations]
ratpac2/src/physics/src/GLG4PMTOpticalModel.cc:501:44: warning: ‘void G4FastStep::SetPrimaryTrackFinalPolarization(const G4ThreeVector&, G4bool)’ is deprecated: use ProposePrimaryTrackFinalPolarization instead [-Wdeprecated-declarations]
ratpac2/src/physics/src/GLG4PMTOpticalModel.cc:510:52: warning: ‘void G4FastStep::SetPrimaryTrackFinalEventBiasingWeight(G4double)’ is deprecated: use ProposePrimaryTrackFinalEventBiasingWeight instead [-Wdeprecated-declarations]
```

Geant4 version 11.2.1

--- a/src/physics/src/GLG4PMTOpticalModel.cc
+++ b/src/physics/src/GLG4PMTOpticalModel.cc
@@ -519,17 +519,10 @@ void GLG4PMTOpticalModel::DoIt(const G4FastTrack &fastTrack, G4FastStep &fastSte
     }
   }
 
-#if G4VERSION_NUMBER > 1130
   fastStep.ProposePrimaryTrackFinalPosition(pos);
   fastStep.ProposePrimaryTrackFinalTime(time);
   fastStep.ProposePrimaryTrackFinalMomentumDirection(dir);
   fastStep.ProposePrimaryTrackFinalPolarization(pol);
-#else
-  fastStep.SetPrimaryTrackFinalPosition(pos);
-  fastStep.SetPrimaryTrackFinalTime(time);
-  fastStep.SetPrimaryTrackFinalMomentum(dir);
-  fastStep.SetPrimaryTrackFinalPolarization(pol);
-#endif
 
   // fastStep.SetPrimaryTrackPathLength( trackLength ); // does anyone care?
   if (weight <= 0) {
@@ -539,11 +532,7 @@ void GLG4PMTOpticalModel::DoIt(const G4FastTrack &fastTrack, G4FastStep &fastSte
     }
   } else {
     // in case multiphoton has been partly absorbed and partly reflected
-#if G4VERSION_NUMBER > 1130
     fastStep.ProposePrimaryTrackFinalEventBiasingWeight(weight);
-#else
-    fastStep.SetPrimaryTrackFinalEventBiasingWeight(weight);
-#endif
   }
 
   if (iloop >= max_iloop) {

--- a/src/physics/src/GLG4PMTOpticalModel.cc
+++ b/src/physics/src/GLG4PMTOpticalModel.cc
@@ -519,10 +519,18 @@ void GLG4PMTOpticalModel::DoIt(const G4FastTrack &fastTrack, G4FastStep &fastSte
     }
   }
 
+#if G4VERSION_NUMBER > 1130
+  fastStep.ProposePrimaryTrackFinalPosition(pos);
+  fastStep.ProposePrimaryTrackFinalTime(time);
+  fastStep.ProposePrimaryTrackFinalMomentumDirection(dir);
+  fastStep.ProposePrimaryTrackFinalPolarization(pol);
+#else
   fastStep.SetPrimaryTrackFinalPosition(pos);
   fastStep.SetPrimaryTrackFinalTime(time);
   fastStep.SetPrimaryTrackFinalMomentum(dir);
   fastStep.SetPrimaryTrackFinalPolarization(pol);
+#endif
+
   // fastStep.SetPrimaryTrackPathLength( trackLength ); // does anyone care?
   if (weight <= 0) {
     fastStep.ProposeTrackStatus(fStopAndKill);
@@ -531,7 +539,11 @@ void GLG4PMTOpticalModel::DoIt(const G4FastTrack &fastTrack, G4FastStep &fastSte
     }
   } else {
     // in case multiphoton has been partly absorbed and partly reflected
+#if G4VERSION_NUMBER > 1130
+    fastStep.ProposePrimaryTrackFinalEventBiasingWeight(weight);
+#else
     fastStep.SetPrimaryTrackFinalEventBiasingWeight(weight);
+#endif
   }
 
   if (iloop >= max_iloop) {

